pypeit/flatfield.py

```diff
@@ -2010,4 +2043,105 @@ def merge(init_cls, merge_cls):
     return FlatImages(**dd)
 
 
+def write_pixflat_to_fits(pixflat_norm_list, detname_list, spec_name, outdir, pixelflat_name, to_cache=True):
```

Comment:
I think we need to create a `SlitlessFlat` class that subclasses from `DataContainer`.

Reply:
I did not managed to created a a SlitlessFlat class that subclasses from DataContainer, but I created a SlitlessFlat class that holds the calculation for generate the slitless flats.